docs/eog/eog-in-practice/cw23/feasibility-stage/fs-outputs-and-outcomes.rst

```diff
@@ -0,0 +1,40 @@
+.. _cw23-fs-outputs-and-outcomes:
+
+CW23 Outputs and Outcomes
+==========================
+
+Outputs are the things produced during or for the workshop, be it physical or virtual, and outcomes are the changes made possible due to running the event.
+These should be linked to the event goals.
+Some of these will be the same as the objectives of the workshop.
+They should include the type of resources that will be available after the event.
+That is, whether a report will be written, videos of the sessions or any other resources will be made available.
+
+
+Outputs
+--------------------
+
+- Recordings of the sessions shared via the SSI Youtube channel
+- Closed captioning, transcriptions
+- Blog posts from the discussion session
+- All slides and other outputs shared via the conference portal (e.g. Figshare)
+- Collaborative Ideas to be recorded via https://github.com/softwaresaved/CW-ideas
+- Hack Day projects
+- A highlights blog post on the SSI website
+- A reflections blog post on the SSI website on how we adapted CW to run as a hybrid event
+- Collaborative notes documents and other unconference infrastructure resources shared
+- Feedback into Event Organisation Guide (e.g. as an In Practice chapter)
+- Any sponsor outputs (e.g. Tweets, videos, etc.)
+- Publish a report on e.g. Zenodo or Figshare detailing our experience of running a hybrid event with lessons learned and a breakdown of where the costs dominate
+
```

Comment:
Some of these won't happen perhaps? or might happen in a different way e.g. the report on Zenodo might be the in practice sections formatted into one doc - but I guess this can be discussed or mentioned in the event project stage 'Outputs and Outcomes Plan'